src/main/webapp/utilities/MapsInterface.js

```diff
@@ -0,0 +1,105 @@
+class MapInterface {
+    /**
+     * @param element the DOM element the map to be rendered
+     * @param centerCoordinates coordinates object that will be the center of the map
+     * @param options (optional) additional parameters that might be needed
+     */
+    constructor(element, centerCoordinates, options = {}) {
+        this.map = new google.maps.Map(element, {
+            center: centerCoordinates,
+            zoom: options.zoom || 8,
+        });
+
+        // Service that fetches the route (used on drawRoute)
```

Comment:
s/on/by/

?

Same below

Reply:
Fixed!

Reply:
https://screenshot.googleplex.com/66ku3cc2LthdDSo Is it?

Reply:
You fixed it below on line 26